src/libs/DistanceRequestUtils.ts

```diff
@@ -1,18 +1,55 @@
-import type {OnyxEntry} from 'react-native-onyx';
+import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
+import Onyx from 'react-native-onyx';
 import type {LocaleContextProps} from '@components/LocaleContextProvider';
+import type {RateAndUnit} from '@src/CONST';
 import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {LastSelectedDistanceRates, Report} from '@src/types/onyx';
 import type {Unit} from '@src/types/onyx/Policy';
 import type Policy from '@src/types/onyx/Policy';
+import type {EmptyObject} from '@src/types/utils/EmptyObject';
 import * as CurrencyUtils from './CurrencyUtils';
 import * as PolicyUtils from './PolicyUtils';
+import * as ReportUtils from './ReportUtils';
 
-type DefaultMileageRate = {
+type MileageRate = {
     customUnitRateID?: string;
     rate?: number;
     currency?: string;
     unit: Unit;
+    name?: string;
 };
 
+const policies: OnyxCollection<Policy> = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.POLICY,
```

Comment:
NAB: Should we use `waitForCollectionCallback` here too?

Reply:
we no longer need it, so I'm removing this